src/commands/query.js

```diff
@@ -23,10 +24,9 @@ class Command {
     wolfram.query(query, (error, result) => {
 
       if (error || result.length === 0) {
-
-        payload.message.reply('I couldn\'t find anything.');
+        var rand = getRandom(0,3);
```

Comment:
You need to import getRandom.

Also remove the variable and plug it directly into the array on the next line.